### PR TITLE
Fix for AKV signing and update docs

### PR DESCRIPTION
**Pull request by beejones**
The PR fixes the submission of proposals signed with an AKV member key. We were missing the voting on proposals.
Small edits to docs.

### CODE REVIEW

**Comment by beejones** [comment]
Testing:
https://github.com/microsoft/privacy-sandbox-dev/actions/runs/11256127843

**Comment by takuro-sato** [comment]
How can KMS members vote to a proposal that were created by another member? Is it future work or already covered by the current code?

File: Makefile [hunk 1]
```diff
@@ -109,15 +109,16 @@ refresh-key: ## 🚀 Refresh a key on the instance
...
-set-constitution: start-host-idp ## Set new custom constitution
+set-constitution: ## Set new custom constitution
 	@echo -e "\e[34m$@\e[0m" || true
 	$(call check_defined, KMS_URL)
 	$(call check_defined, KEYS_DIR)
-	 if ! cmp -s ${KMS_WORKSPACE}/sandbox_common ${KEYS_DIR}; then \
-	 	@sleep 5
+	if [ "${KMS_WORKSPACE}/sandbox_common" != "${KEYS_DIR}" ]; then \
+		echo "Copying files for constitution"; \
+		@sleep 5; \
```
Comment (takuro-sato) [inline Makefile:118]: That's great you clarified the purpose.
Can we avoid using sleep or make an issue for that? I faced 'missing constitution' errors a lot in the past, maybe this is the reason.

Comment (beejones) [reply Makefile:118]: correct. We needed to copy these files manually. Now it will be automatically

Comment (beejones) [reply Makefile:118]: There us a comment in the AKV signing script that only one member signs the proposal. We need an additional script only for the voting. I created an issue:
https://github.com/microsoft/azure-privacy-sandbox-kms/issues/156

**Review by takuro-sato — APPROVED** [review]

**Pull request merged**